package.json

```diff
@@ -130,5 +130,9 @@
     "rollup-plugin-visualizer": "^4.0.4",
     "type-fest": "^0.21.0",
     "typescript": "^4.1.2"
+  },
+  "resolutions": {
+    "@types/react": "^16.9.41",
+    "@types/react-dom": "^16.9.8"
```

Comment:
@prateek3255 I'm just stumbling upon this and I don't really know what this does or what this is for. I must have missed this when we merged this. Can you explain what this does or why we need it? And do we need to update it for v4 to react18 or something? Thanks!

Reply:
@TkDodo From what I remember this was added because there was some bunch of type errors when adding DOM related types to devtools and its utils probably because of some conflicting `@types/react` of some other package. So adding resolutions helped resolve to the type version we wanted to use.

From what I see now is that we are not getting any type errors while compiling both on master and alpha if we remove this, so this can just be safely removed.